gitbook-docs/data_model.md

```diff
@@ -263,6 +263,32 @@ the value should be merged to the full model mounted where the delta‘s context
   ]
 }
 ```
+## Delta Format For Metadata
+
+Metadata can also be specified via a delta. The delta is very similiar to the `values` deltas above, but instead of having `values` key, it will have a `meta` key. Note that a client could multiple meta deltas for any given path from different sources, so the client merge the meta information.
```

Comment:
Check grammar. Also clarify that the full meta must be provided in the meta delta (ie. the same as values where the full value must be supplied).

What does merge metas refer to? There should be one reliable source of meta at the Master Main.

Also we need an explanation - probably in subscription_protocol.md as to when these meta-deltas are required to be sent by the server and the relationship to different subscription policies. I'm assuming that meta-delta will only be sent on an 'instant' basis to existing subscriptions even if they are not 'instant'??

Reply:
see PR #491 for suggested changes